src/utils/movieUtils.js

```diff
@@ -9,7 +9,15 @@ const getReleaseYear = (release_date) => {
     return (new Date(release_date).getFullYear())
 }
 
+const fetchMovies = async (url) => {
+    const response = await fetch(url);
+    const resData = await response.json();
+
+    return resData
+}
```

Comment:
```suggestion
const fetchMovies = async (url) => {
    const response = await fetch(url);
    const resData = await response.json();

    return resData.data
}
```

Reply:
@MykhailoSheliahov it's universal function for the get movie by id and get movie list. When get movie by id it returns already object for movie, when we get list - it returns object with additional data and also array of movies.

Reply:
@SerhiiSmilianets should we create 2 function  fetchMovie and fetchMovies to separate logic?